dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -90,6 +90,10 @@ public class MetadataDumper {
 
   private boolean exitOnError = false;
 
+  private int totalNumberOfTasks = 0;
+  private int numberOfCompletedTasks = 0;
+  long startTimeInMilliseconds = 0;
```

Comment:
I would suggest using `StopWatch` instead of `long` for measuring time.

See the example usage:
https://github.com/google/dwh-migration-tools/blob/412897cf34fa90355783fb7f27523948cab32a90/dumper/lib-common/src/main/java/com/google/edwmigration/dumper/plugin/ext/jdk/progress/RecordProgressMonitor.java#L40